### PR TITLE
Add kudos to appdata files

### PR DESCRIPTION
**Pull request by mihkelvain**
This changes is requested by Richard Hughes, appdata maintainer for GNOME/Fedora. We get more stars in software centers this way

### CODE REVIEW

File: qesteidutil.appdata.xml [hunk 1]
```diff
@@ -29,5 +29,9 @@
     <screenshot type="default">http://mihkel.fedorapeople.org/esteid/qesteidutil.png</screenshot>
   </screenshots>
   <url type="homepage">http://www.id.ee/</url>
+  <kudos>
```
Comment (martinpaljak) [inline qesteidutil.appdata.xml]: What is the purpose of these lines ?

**Comment by mihkelvain** [comment]
Hi.
As said, those kudos lines will get us more stars in gnome software center and possibly in other software centers too. The reason Richard asked me to add these is that esteid packages in Fedora are in good shape (eg. worth a lot of stars or kudos as they call), but their rating system is being re-designed.

Snippet from mail:
I'm emailing you directly as you've been listed as the update contact in one or more AppData files. In future versions of the appstream-builder (the program used by distributions to merge all the AppData files in packages into AppStream metadata) we're going to be running a lot less crazy plugins. At the moment we're doing things like parsing .gir files, running nm on binaries and looking up things from arrays of hardcoded data just so we can try and work out what kudos the application deserves. This doesn't scale, and means the machine generating the metadata needs to have all kinds of extra build tools installed.

Of course if you don't like to merge those changed upstream, I can make a Fedora specific patch, but I am lazy and don't like to do extra work :)

**Comment by mihkelvain** [comment]
Explanation:
https://git.gnome.org/browse/gnome-software/plain/doc/kudos.md

**Comment by martinpaljak** [comment]
OK, the link explains it. Seems to make sense (for non-Fedoras as well)

**Comment by metsma** [comment]
Merged

**Pull request closed without merging**